docs/start_nominating.md

```diff
@@ -123,12 +123,34 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
 
 <div style={{textAlign: 'center'}}>
   <img src={useBaseUrl('/nominator-guide/nominate-validator-4.png')} />
-</div>
+</div>  
```

Comment:
```suggestion
</div>
```

Reply:
Double-space is a linebreak in markdown (<br>)